pkg/internal/testhelpers/containers_test.go

```diff
@@ -46,7 +46,7 @@ func TestWithDB(t *testing.T) {
 	if testing.Short() {
```

Comment:
With this test setup, there is no need for checking the short flag, since the integration tests are in a separate package.